morpheus/utils/onnx_to_trt.py

```diff
@@ -27,7 +32,12 @@ def gen_engine(c: ConfigOnnxToTRT):
     """
 
     # Local imports to avoid requiring TensorRT to generate the docs
-    import tensorrt as trt
+    try:
+        import tensorrt as trt
+    except:
+        logger.error("onnx_to_trt requires the tensorrt module and runtime to be installed.")
+        return
+
```

Comment:
Can you move this to the top of the file? It doesnt need to be inside the function. You should add
```python
autodoc_mock_imports = [
    "morpheus.cli",  # Dont document the CLI in Sphinx
    "tqdm",
    "tensorrt",  # Ignore loading tensorrt
]
```
to `docs/source/conf.py` to prevent this from being loaded as part of the sphinx autodoc